barcalendar.py

```diff
@@ -534,7 +536,7 @@ def parse_version_name(line):
 python_releases = [
     # Version, and Year-Month for start and end of support.
     #('2.7', 2010, 7, 2019, 12),
-    ('3.5', 2015, 9, 2020, 9),          # https://www.python.org/dev/peps/pep-0478/
+    # ('3.5', 2015, 9, 2020, 9),          # https://www.python.org/dev/peps/pep-0478/
```

Comment:
Do you know why we comment put rather than deleting?

Reply:
We never actually decided on how to keep the old entries. I am thinking of doing another follow up optimization of the script so we can handle this later on.